### ISSUE TITLE
Strange Alias -> page not found

### ISSUE THREAD

**Issue by MeiRos** [owncloud/core#23540]
### Steps to reproduce
1. Do fresh install
wget -qO - https://download.owncloud.org/download/repositories/stable/xUbuntu_14.04/Release.key | sudo apt-key add -

echo 'deb http://download.owncloud.org/download/repositories/stable/xUbuntu_14.04/ /' | sudo tee /etc/apt/sources.list.d/owncloud.list

apt-get update
apt-get install owncloud

2. Go to webbrowser and type yourserver/owncloud or yourserver/owncloud/index.php
3. You'll get page not found

### Expected behaviour
I should be able to configure owncloud admin account and other settings. 

### Actual behaviour
Page not found.

### Server configuration
**Operating system**:
Ubuntu 14.04.4
**Web server:**
Apache 2.4.7
**Database:**
Mysql 14.14 distrib 5.5.47
**PHP version:**
5.5.9-1ubuntu4.14
**ownCloud version:** (see ownCloud admin page)
9.0.0 (stable)

**Updated from an older ownCloud or fresh install:**
Fresh
**Where did you install ownCloud from:**
Read reproduce
**
```

My solution to fix this.
Edit owncloud.conf in /etc/apache2/conf-available. 
Change Alias /owncloud /usr/share/owncloud --> Alias /owncloud /var/www/owncloud

Default document root is /var/www/html and owncloud was installed /var/www/owncloud.

**Comment by LukasReschke** [comment]
@jnweiger

**Comment by jnweiger** [comment]
The alias you saw was clearly wrong. I don't think this was a fresh install.
Wit the new owncloud-files package we bring in our config template as /usr/share/doc/owncloud-files/owncloud-config-apache.conf.default

with ubuntu14.04 the dependency setup is done by package owncloud-deps-php5, which copies the default to /etc/apache2/conf-available/owncloud.conf unless(!) there is already such a file there.

Furthermore, symlinks to conf-enabled are added and removed as approproate.

I assume, you did not really have a fresh install, you had an older (and broken) /etc/apache2/conf-available/owncloud.conf in the system, which just got enabled. We don't overwrite existing config files during install.

**Comment by MeiRos** [comment]
Well, this is quite weird then. This was my first time ever install owncloud and my vps was setup about week ago, so there haven't been owncloud before. If owncloud.conf only comes with owncloud, I really don't know why the old file was there. But, because I haven't used owncloud ever before there's possibility I have messed something. I just don't know what...

**Comment by RealRancor** [comment]
@jnweiger Not directly related to the ``Alias /owncloud /usr/share/owncloud```but in owncloud-files_9.0.0-1.1_all.deb from the repository linked above two wrong references to:

```
  SetEnv HOME /usr/share/owncloud
  SetEnv HTTP_HOME /usr/share/owncloud
```

can be found.

**Comment by jnweiger** [comment]
@RealRancor yes, known and also fixed in 9.0.1
Please reopen, if reproducable.
